src/Styles/BookItem.css

```diff
@@ -0,0 +1,191 @@
+  font-size: 0.875rem;
+  font-weight: bold;
+  color: var(--black-two);
+}
+
+.book-title {
+  font-family: 'RobotoSlab Bold', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  color: var(--black-two);
+}
+
+.book-actions {
+  margin-top: 1.313rem;
+  display: flex;
+}
+
+.book-author,
+.book-actions button {
+  font-family: 'RobotoSlab Light', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #0290ff;
+}
+
+.book-actions button {
+  background-color: transparent;
+  border: none;
+  padding: 0;
+  margin: 0;
+  cursor: pointer;
+}
+
+.book-actions button:not(:last-child)::after {
+  content: '|';
+  margin: 0 0.938rem;
+  line-height: 1.5rem;
+  color: var(--white-two);
+}
+
+.book-progress {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.progress-stats {
+  display: flex;
+  flex-direction: column;
+  margin-left: 1.313rem;
+}
+
+.percentage {
+  font-family: 'Montserrat', serif;
+  font-size: 2rem;
+  font-weight: normal;
+  color: var(--black-two);
+}
+
+.percentage-completed {
+  opacity: 0.5;
+  font-family: 'Montserrat', serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  color: var(--black-two);
+}
+
+.current-chapter {
+  opacity: 0.5;
+  font-family: 'RobotoSlab Light', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  color: var(--black-two);
+}
+
+.chapter {
+  font-family: 'RobotoSlab Light', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+  color: var(--black-two);
+}
+
+.chapter-info {
+  border-left: solid 1px var(--white-two);
+  padding-left: 4%;
+  margin-left: -4%;
+}
+
+.update-progress {
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  font-family: 'RobotoSlab Light', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: white;
+  border: none;
+  margin-top: 1.438rem;
+  width: 11.5rem;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  width: 75%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 7vh;
+}
+
+.form form {
+  display: flex;
+  flex-direction: row;
+}
+
+.add-book-title {
+  font-family: 'Montserrat', serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  letter-spacing: -0.18px;
+  color: var(--warm-grey);
+  margin-bottom: 1.888rem;
+}
+
+.form input {
+  height: 2.813rem;
+  border-radius: 4px;
+  border: solid 1px var(--white-two);
+  background-color: white;
+  width: 40%;
+  margin-right: 2.125rem;
+  padding: 0 1.063rem;
+}
+
+.form input::placeholder {
+  font-family: 'Montserrat', serif;
+  font-size: 1rem;
+  letter-spacing: -0.15px;
+  color: black;
+}
+
+.add-book-btn {
+  width: 11.5rem;
+  height: 2.813rem;
+  border-radius: 3px;
+  border: none;
+  background-color: #0290ff;
+  font-family: 'RobotoSlab Bold', serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: white;
+  cursor: pointer;
+}
```

Comment:
- Let's use the correct font to make our design look close to the main design:

*main design*
![image](https://github.com/mdg0410/Bookstore_react/assets/59397159/0866baea-24ea-4468-a439-35bc811e7f7d)



*Your design*
![image](https://github.com/mdg0410/Bookstore_react/assets/59397159/a0d4dd8d-51ea-41b6-add9-ae49d141603b)

Reply:
> Correction 

## Import font-family: 
- Montserrat
- Roboto Slab

![imagen](https://github.com/mdg0410/Bookstore_react/assets/90431831/bb7ce9ea-189e-4848-87f1-a99064731b52)